NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -62,6 +62,16 @@ public enum NeuroID {
 
     internal static var isRN: Bool = false
     internal static var rnOptions: [RNConfigOptions: Any] = [:]
+    
+    internal static var NID_ORIGIN_NID_SET = "nid"
+    internal static var NID_ORIGIN_CUSTOMER_SET = "customer"
+    internal static var NID_ORIGIN_CODE_FAIL = "400"
+    internal static var NID_ORIGIN_CODE_NID = "200"
+    internal static var NID_ORIGIN_CODE_CUSTOMER = "201"
```

Comment:
if these are consts and not changeable properties can we put them in the constants file?